### DISCUSSION TITLE
Power Platform CLI: pac admin list-settings / update-settings

### DISCUSSION THREAD

**Discussion by Laskewitz** [microsoft/powerplatform-build-tools#460] [Ideas]
I would like to have commands available to change tenant settings, like the ones that are available already with PowerShell (get/set-tenantsettings).

# Example: 

- `pac admin update-settings --key <key> --value <value>` 
- `pac admin update-settings --input-file ./settings.csv/json`

# Link to tenant settings PowerShell
https://learn.microsoft.com/en-us/power-platform/admin/powerapps-powershell#display-tenant-setting-for-ability-to-share-apps-with-everyone

**Comment by anttipajunen** [comment]
Another great idea!

**Comment by Laskewitz** [comment]
@petrochuk - FYI, these will also help with copilot settings, because there are some copilot settings in there.

<img width="612" alt="image" src="https://github.com/microsoft/powerplatform-build-tools/assets/3766843/7d37d219-d905-4301-902c-a5cd3a8f306d">

**Comment by snizar007** [comment]
Thank you @Laskewitz, we will look into this feedback.

**Comment by petrochuk** [comment]
https://learn.microsoft.com/en-us/power-platform/developer/cli/reference/admin#pac-admin-list-tenant-settings
